Servers/UI/OJS.Servers.Ui/ClientApp/src/common/types.ts

```diff
@@ -90,7 +90,7 @@ interface IStartParticipationResponseType {
     contestIsCompete: boolean;
     lastSubmissionTime: Date;
     remainingTimeInMilliseconds: number;
-    validationResult: IStartParticipationValidationType;
+    validationResult: IValidationType;
```

Comment:
move this either in the begginning or end of interface members so its easier to read

Reply:
Done.